File: backend/art-galery/src/main/java/com/unmsm/movil/tecnisis/art_galery/infrastructure/adapters/input/rest/model/response/SpecialistResponse.java
package com.unmsm.movil.tecnisis.art_galery.infrastructure.adapters.input.rest.model.response;

import lombok.*;

import java.util.List;

@Builder @Getter @Setter
@NoArgsConstructor
@AllArgsConstructor
public class SpecialistResponse {
    private Long id;
    private Boolean isAvailable;
    private PersonResponse person;
    List<TechniqueResponse> techniques;
}

File: tecnisis-app/app/src/main/java/com/example/tecnisis/data/specialist/SpecialistResponse.kt
package com.example.tecnisis.data.specialist

import com.example.tecnisis.data.person.PersonResponse
import com.google.gson.annotations.SerializedName

data class SpecialistResponse(
    val id: Long,
    val person: PersonResponse,
    @SerializedName("is_available")
    val isAvailable: Boolean
)
